types/patches/0001-Trivial-patch-as-placeholder.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Greg Price <greg@zulip.com>
+Date: Tue, 31 May 2022 20:35:47 -0700
+Subject: [PATCH] Trivial patch, as placeholder
+
+---
+ types/@react-navigation/stack.js.flow | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/types/@react-navigation/stack.js.flow b/types/@react-navigation/stack.js.flow
+index 253b2f62f..1e60c5cf4 100644
+--- a/types/@react-navigation/stack.js.flow
++++ b/types/@react-navigation/stack.js.flow
```

Comment:
```
tools/tsflower: Script to run TsFlower and manage patches on output
```

As I understand it, these patches are meant to bridge the gap between TsFlower's reasonable translation of the TypeScript, and some better types that weren't expressed in the TypeScript (and maybe couldn't have been).

Could you write a bit about that process, and how `tools/tsflower` is meant to be used to help it along? Maybe in the output of `tools/tsflower --help`.

Reply:
Yeah, I should definitely give the script a usage message and describe the workflows.

For an idea of what the patches are about, there's a larger set of them in #5397. The PR description has (copied from the main commit message) a summary of the several different purposes they serve.

The majority of them are just as you mentioned; there are also some for things that could be fixed in TsFlower. Basically I got TsFlower to the point where the number of errors was small enough that I figured it made more sense to switch to writing this script and those patches and getting it shipped, after which I can fix remaining TsFlower issues one by one at leisure. In steady state I'd aim for the "TsFlower issue" category of patch to largely disappear.